witchcraft/witchcraft.go

```diff
@@ -601,17 +603,42 @@ func (s *Server) Start() (rErr error) {
 	ctx = metrics.WithRegistry(ctx, metricsRegistry)
 
 	// initialize loggers
-	s.initLoggers(baseInstallCfg.UseConsoleLog, wlog.InfoLevel, metricsRegistry)
-
-	// add loggers to context
-	ctx = s.withLoggers(ctx)
```

Comment:
Is there any logic that logs on the context loggers in `s.initRuntimeConfig`? That would be impacted by the removal here.

Reply:
Good catch! The `s.initRuntimeConfig` does need the loggers on the context, so I moved it back to where it was. I also looked into what `s.withLoggers` is doing and it appears safe to re-run after we re-initialize the loggers for the TCP writer, so I added an additional call below.